src/main/java/frc/team2412/robot/commands/intake/AllInCommand.java

```diff
@@ -20,7 +20,7 @@ public void initialize() {
 
 	@Override
 	public void execute() {
-		if (intakeSubsystem.getIndexSensor()) {
+		if (intakeSubsystem.getIndexSensor() && !intakeSubsystem.getSensorOverride()) {
```

Comment:
To make the logic more clear, is it better to override the sensor here or in the getSensorOverride method?
Or with a parameter to getSensorOverride that overrides the sensor?